source/templates/4-Action-Set.json

```diff
@@ -47,7 +47,7 @@
                     "resources": [
                         {
                             "condition": "[equals(toLower(parameters('type')), toLower('Microsoft.Compute/virtualMachines'))]",
-                            "apiVersion": "2018-11-20",
```

Comment:
Same here. I think we should use the same updated apiVersion for HybridCompute as well